.gitignore

```diff
@@ -16,6 +16,7 @@
 node_modules
 
 *.log
+db/*
```

Comment:
You won't be able to commit the dir. You need to put .gitigore inside. Look at `logs` dir